src/test/java/com/google/api/codegen/config/LongRunningConfigTest.java

```diff
@@ -35,8 +35,8 @@
   private static final String GAPIC_CONFIG_METADATA_TYPE = "HeaderType";
   private static final String ANNOTATIONS_RETURN_TYPE_NAME = "BookType";
   private static final String ANNOTATIONS_METADATA_TYPE = "FooterType";
-  private static final boolean TEST_IMPLEMENTS_DELETE = false;
-  private static final boolean TEST_IMPLEMENTS_CANCEL = false;
+  private static boolean TEST_IMPLEMENTS_DELETE = false;
```

Comment:
Why is the `final` being removed here?

Reply:
Oops. Making all those private static primitives final now.